client/state/jetpack-settings/modules/test/actions.js

```diff
@@ -0,0 +1,88 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+import sinon from 'sinon';
+
+/**
+ * Internal dependencies
+ */
+import {
+	activateModule
+} from '../actions';
+
+import {
+	JETPACK_MODULE_ACTIVATE,
+	JETPACK_MODULE_ACTIVATE_SUCCESS,
+	JETPACK_MODULE_ACTIVATE_FAILURE
+} from 'state/action-types';
+import useNock from 'test/helpers/use-nock';
+
+describe( 'actions', () => {
+	const spy = sinon.spy();
+
+	beforeEach( () => {
+		spy.reset();
+	} );
+	describe( '#activateJetpackModule', () => {
+		const siteId = 123456;
+
+		useNock( ( nock ) => {
+			nock( 'https://public-api.wordpress.com:443' )
+			.post( '/rest/v1.1/sites/123456/jetpack/modules/module-a' )
+			.reply( 200, {} );
+		} );
+
+		it( 'should dispatch JETPACK_MODULE_ACTIVATE when trying to activate a module', () => {
+			activateModule( siteId, 'module-a' )( spy );
+
+			expect( spy ).to.have.been.calledWith( {
+				type: JETPACK_MODULE_ACTIVATE,
+				siteId,
+				moduleSlug: 'module-a'
+			} );
+		} );
+	} );
+	describe( '#activateJetpackModuleSuccess', () => {
```

Comment:
I'd wrap all tests for an action within the `describe()` call of the action, instead of just creating another for each case. So the structure of the tests would become like:

- actions
  - activateJetpackModule
    - success
      - `JETPACK_MODULE_ACTIVATE`
      - `JETPACK_MODULE_ACTIVATE_SUCCESS`
    - failure
      - `JETPACK_MODULE_ACTIVATE_FAILURE`

or

- actions
  - activateJetpackModule
    - `JETPACK_MODULE_ACTIVATE`
    - success
      - `JETPACK_MODULE_ACTIVATE_SUCCESS`
    - failure
      - `JETPACK_MODULE_ACTIVATE_FAILURE`